### PR TITLE
[WebDriver] Add test for triple click

### PR DESCRIPTION
**Pull request by AutomatedTester**
This test does a triple-click and then checks that the entire paragraph
has been selected.

This currently fails in all drivers.

### CODE REVIEW

File: webdriver/tests/perform_actions/pointer_tripleclick.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    div = session.find.css("div", all=False)
+    div_rect = div.rect
+    div_centre = {
+        "x": div_rect["x"] + div_rect["width"]/2,
+        "y": div_rect["y"] + div_rect["height"]/2
+    }
+    mouse_chain \
+        .pointer_move(div_centre["x"], div_centre["y"]) \
+        .click() \
+        .pause(0) \
```
Comment (jgraham) [inline webdriver/tests/perform_actions/pointer_tripleclick.py:23]: What are the pauses for here?

File: webdriver/tests/perform_actions/pointer_tripleclick.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        .click() \
+        .pause(0) \
+        .click() \
+        .perform()
+
+    actual_text = session.execute_script("return document.getSelection().toString();")
```
Comment (jgraham) [inline webdriver/tests/perform_actions/pointer_tripleclick.py:29]: This behaviour seems platform-specific. I'm not sure how else to test though; it seems like double-click has its own event but not triple.

Comment (AutomatedTester) [reply webdriver/tests/perform_actions/pointer_tripleclick.py:29]: From what I can find, all browsers have gone to triple-click means select a paragraph. I can't find anything that suggests it is platform specific. And yes, there is no event I can watch for.

File: webdriver/tests/perform_actions/pointer_tripleclick.py (hunk 1 above)
Comment (AutomatedTester) [reply webdriver/tests/perform_actions/pointer_tripleclick.py]: I copied the double click test. I can't see any reason why they should be there for this test so have removed them

File: webdriver/tests/perform_actions/pointer_tripleclick.py (hunk 2 above)
Comment (jgraham) [reply webdriver/tests/perform_actions/pointer_tripleclick.py:27]: What about mobile? At least triple-tap doesn't seem to work the same there. So I think there should at least be a comment indicating that the test might be making nonstandard assumptions to help out people who are investigating failures in the future.

**Review by jgraham — CHANGES_REQUESTED** [review]

File: webdriver/tests/perform_actions/pointer_tripleclick.py (hunk 2 above)
Comment (AutomatedTester) [reply webdriver/tests/perform_actions/pointer_tripleclick.py:32]: comment added

**Review by jgraham — APPROVED** [review]

**Comment by PlayerMPark** [comment]
How do i run this? I get ImportError: No module named tests.perform_actions.support.refine

**Comment by PlayerMPark** [comment]
> This test does a triple-click and then checks that the entire paragraph
> has been selected.
> 
> This currently fails in all drivers.

**Comment by whimboo** [comment]
Here the most recent results:

https://wpt.fyi/results/webdriver/tests/perform_actions/pointer_tripleclick.py?label=master&label=experimental&product=chrome&product=firefox&product=safari

@AutomatedTester can you please have a look?

**Pull request merged**